scripts/has-sourcemap-debugid.js

```diff
@@ -0,0 +1,31 @@
+const process = require('process');
+const fs = require('fs');
+
+const sourceMapPath = process.argv[2];
+
+if (!sourceMapPath) {
+  console.log('Add source map path as first argument of the script.');
```

Comment:
should we `console.warn` or `console.error` these? Ditto for the console logs that are below this.

Reply:
I agree that's more correct, but I had issues reading std err output in https://github.com/getsentry/sentry-react-native/pull/3164/files#diff-01c94c11105e37de1b11a82973b605f625fff5959b0386c32fd1e42a14325208R125-R126

So that's why I kept it `console.log`. More important is that we have the log.